x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -461,3 +461,5 @@ export const RISKY_HOSTS_DOC_LINK =
   'https://www.elastic.co/guide/en/security/current/host-risk-score.html';
 export const RISKY_USERS_DOC_LINK =
   'https://www.elastic.co/guide/en/security/current/user-risk-score.html';
+
+export const MAX_LIMIT_TIMELINE = 2000;
```

Comment:
would you mind renaming this to `BULK_ADD_TO_TIMELINE_MAX` or something that notes it's specific for bulk add to timeline

Reply:
done.